FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/SchedulerService.cs

```diff
@@ -40,20 +44,23 @@ public class SchedulerService : ISchedulerService
             IQueueClient queueClient,
             IMetadataStore metadataStore,
             IOptions<JobConfiguration> jobConfiguration,
+            IMetricsLogger metricsLogger,
```

Comment:
just curios when we use error processor and when use metriclogger. seems a little messy here

Reply:
Using error processors if catch exception. 
metricsLogger is removed in this file.